etna/models/catboost.py

```diff
@@ -283,6 +283,8 @@ def __init__(
             * For GPU. The given value is used for reading the data from the hard drive and does
               not affect the training.
               During the training one main thread and one thread for each GPU are used.
+        **kwargs:
+            Additional parameters passed to `CatBoostRegressor` model.
```

Comment:
I think we should write:
```
Additional parameters passed to :py:class:`catboost.CatBoostRegressor` model.
```
It will point out that it is external library.